plugins/toolkit/jetbrains-core/tst/software/aws/toolkits/jetbrains/core/explorer/AwsToolkitExplorerToolWindowTest.kt

```diff
@@ -40,22 +40,23 @@ class AwsToolkitExplorerToolWindowTest {
         (ToolWindowManager.getInstance(projectExtension.project) as ToolWindowHeadlessManagerImpl)
             .doRegisterToolWindow(AwsToolkitExplorerFactory.TOOLWINDOW_ID)
         val sut = runInEdtAndGet { AwsToolkitExplorerToolWindow(projectExtension.project) }
+        runInEdtAndGet {
+            sut.loadState(
+                AwsToolkitExplorerToolWindowState().apply {
+                    selectedTab =
+                        AwsToolkitExplorerToolWindow.EXPLORER_TAB_ID
+                }
+            )
+            assertThat(sut.state.selectedTab).isEqualTo(AwsToolkitExplorerToolWindow.Q_TAB_ID)
 
-        sut.loadState(
-            AwsToolkitExplorerToolWindowState().apply {
-                selectedTab =
-                    AwsToolkitExplorerToolWindow.EXPLORER_TAB_ID
-            }
-        )
-        assertThat(sut.state.selectedTab).isEqualTo(AwsToolkitExplorerToolWindow.Q_TAB_ID)
-
-        sut.loadState(
-            AwsToolkitExplorerToolWindowState().apply {
-                selectedTab =
-                    AwsToolkitExplorerToolWindow.DEVTOOLS_TAB_ID
-            }
-        )
-        assertThat(sut.state.selectedTab).isEqualTo(AwsToolkitExplorerToolWindow.Q_TAB_ID)
+            sut.loadState(
+                AwsToolkitExplorerToolWindowState().apply {
+                    selectedTab =
+                        AwsToolkitExplorerToolWindow.DEVTOOLS_TAB_ID
+                }
+            )
+            assertThat(sut.state.selectedTab).isEqualTo(AwsToolkitExplorerToolWindow.Q_TAB_ID)
```

Comment:
Should this test change now that user can dismiss the Q tab?

Reply:
In this we are just testing whether Q_TAB is selected or not. Actually this test passes but sometimes fails on CI and local. So, running this test in  `runInEdtAndGet {}  ` block. To avoid flakiness.
> Should this test change now that user can dismiss the Q tab?

I think this is an another test case, isn't it?

Reply:
per discussion with Ankit, leave default to be Q_TAB

Reply:
Default tab selected should be Q_TAB, checked with @ankit-sharma90